project/Publish.scala

```diff
@@ -16,14 +16,15 @@ object Publish extends AutoPlugin {
   override def trigger = allRequirements
 
   override lazy val projectSettings = Seq(
-    crossPaths := false,
```

Comment:
did you try a release dry-run?
does the "release" directory match the expected location for rsync and such (in release sh script)

for example we have `crossTarget.value / "release"`

Reply:
Yes, that's right, good catch. I was thinking about it the wrong way. Now fixed and double checked with a release dry-run.